src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,101 @@
+/* Outter Grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+@media (--tablet-and-down) {
+  .bleedBackground {
+    grid-row: 1;
+    grid-template-rows: auto;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
+  row-gap: 50px;
```

Comment:
This `row-gap` doesn't do anything because there's only one row in this grid, so you should delete it.

But you should also delete it as well as pretty much every other grid-related property on this element if you follow my earlier suggestion of combining the `gridParent` and the `gridAreaMiddle` `<div>` elements into a single one.

To be a bit more concrete, could you combine the `gridParent` and `gridAreaMiddle` classes, name it `carouselWrapper`, and delete every property except for `grid-column: 2`? In the HTML, you should also combine the `<div>` elements into one and also name its class to match.

Reply:
👍 Deleted row-gap and combined gridParent and gridAreaMiddle into carouselWrapper.